.github/workflows/_android-build-test.yml

```diff
@@ -23,14 +23,19 @@ on:
         type: string
         description: |
           A JSON description of what configs to run later on.
+      runner:
+        required: false
+        type: string
+        default: "linux.large"
+        description: Runner type
 
 env:
   GIT_DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
 
 jobs:
   filter:
     if: github.repository_owner == 'pytorch'
-    runs-on: [self-hosted, linux.large]
+    runs-on: ${{ inputs.runner }}
```

Comment:
why drop the `self-hosted` label?

Reply:
Two reasons:
* The `[self-hosted, ${{ inputs.runner }}]` syntax does not work.
* We use special labels - this means that runners are self-hosted. Therefore, we can skip the `self-hosted` label.